doc/directives.md

```diff
@@ -0,0 +1,117 @@
+Dartdoc supports several **directives** within Dart doc comments. Each directive
+is then processed during documentation generation, and new text is inserted in
+place of the directive. Not all directives are supported for package
+documentation at https://pub.dev/.
+
+A doc comment directive consists of either one directive tag, which looks
+something like `{@DIRECTIVE ARG=VALUE ...}`, or two such tags (an opening and a
+closing tag), with content in between.
+
+The supported directives are listed below:
+
+## `@nodoc` - Do not include documentation
```

Comment:
Should this have `{` and `}` for consistency with what's below, or is this a special case (in which case we should call that out explicitly)?